build.py

```diff
@@ -69,9 +69,9 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    "2.48.0dev": (
-        "24.06dev",  # triton container
-        "24.06",  # upstream container
+    "2.49.0dev": (
+        "24.07",  # triton container
```

Comment:
I think this becomes 24.08dev? (not sure)

Reply:
Thanks @fpetrini15 your suggestion looks correct. Have incorporated it.